### DISCUSSION TITLE
Emojis and pictures aren't showing

### DISCUSSION THREAD

**Discussion by DafnaPundak** [community/community#22812] [Other Feature Feedback, Questions, & Ideas]
<p>Hi,<br>
I’m not seeing the emojis and picture in the website although I see it perfectly in the preview.<br>
Can I have some help please?</p>

**Comment by otter-computer** [comment]
<p>Hi <a class="mention" href="/dafnapundak">@DafnaPundak</a>, welcome to the GitHub Support Community!</p>
<p>Can you share a link to your site/repository that hosts your site so we can see what’s going wrong?</p>

**Comment by DafnaPundak** [comment]
<aside class="onebox allowlistedgeneric">
  <header class="source">
      <a href="https://dafnapundak.github.io/markdown-portfolio/" target="_blank" rel="noopener nofollow ugc">Welcome to my portfolio!</a>
  </header>
  <article class="onebox-body">
    <img src="" class="thumbnail" width="" height="">

<h3><a href="https://dafnapundak.github.io/markdown-portfolio/" target="_blank" rel="noopener nofollow ugc">Welcome to my portfolio!</a></h3>



  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>

<aside class="onebox allowlistedgeneric">
  <header class="source">
      <img src="https://github.githubassets.com/favicons/favicon.svg" class="site-icon" width="32" height="32">
      <a href="https://github.com/DafnaPundak/markdown-portfolio/tree/master/_includes" target="_blank" rel="noopener nofollow ugc">GitHub</a>
  </header>
  <article class="onebox-body">
    <img src="https://user-images.githubusercontent.com/57066118/181073209-3b443488-7bd4-4266-ac2d-1787720d8944.jpeg" class="thumbnail onebox-avatar" width="400" height="400">

<h3><a href="https://github.com/DafnaPundak/markdown-portfolio/tree/master/_includes" target="_blank" rel="noopener nofollow ugc">DafnaPundak/markdown-portfolio</a></h3>

<p>Contribute to DafnaPundak/markdown-portfolio development by creating an account on GitHub.</p>


  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>

<p>Thank You!</p>

**Comment by otter-computer** [comment, marked as answer]
<p>Thanks for dropping those links!</p>
<p>For your images, it looks like you’ve put them inside the <code>_includes</code> directory. This is a special Jekyll directory that’s intended to be used for storing HTML template fragments or snippets that will be used in multiple locations, not really for storing your site’s assets. I’d recommend moving your image files either out to the root of your repository or moving them to an <code>images</code> directory instead.</p>
<p>Be sure to update the asset links to your image files too! For example, I see that you’ve referenced the image <code>profilepic.png</code> inside this file:</p>
<aside class="onebox githubblob">
  <header class="source">
      <a href="https://github.com/DafnaPundak/markdown-portfolio/blob/17bac0a97575659289b27a5a68f6a7dfe8946cd1/_includes/02-image.md" target="_blank" rel="noopener">github.com</a>
  </header>
  <article class="onebox-body">
    <h4><a href="https://github.com/DafnaPundak/markdown-portfolio/blob/17bac0a97575659289b27a5a68f6a7dfe8946cd1/_includes/02-image.md" target="_blank" rel="noopener">DafnaPundak/markdown-portfolio/blob/17bac0a97575659289b27a5a68f6a7dfe8946cd1/_includes/02-image.md</a></h4>
<pre><code class="lang-md">![this is me](./profilepic.png "this is me!")
</code></pre>


  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>

<p>If you move that image into a directory called <code>images</code> you’ll need to update this link to the following:</p>
<pre><code class="lang-auto">![this is me](images/profilepic.png "this is me!")
</code></pre>
<p>As for the emoji not showing up, emoji conversion from text like <code>:wave:</code> isn’t enabled by default for Pages sites. You need to manually enable a plugin inside your <code>_config.yml</code> file for this feature to work. To do this, add the following to a new line on the end of your <code>_config.yml</code> file:</p>
<pre><code class="lang-auto">plugins:
  - jemoji
</code></pre>
